content/en/blog/2025/sail-operator-ga/index.md

```diff
@@ -0,0 +1,301 @@
+
+### Deploy sample application
+
+- Create a namespace and label it to enable istio injection
+
+{{< text bash >}}
+$ kubectl create namespace sample
+$ kubectl label namespace sample istio-injection=enabled
+{{< /text >}}
+
+After labeling the namespace you will see that the `IstioRevisionTag` resource status will change to In Use true, this is because there is now a resource using the revision `default-v1-24-2`.
+
+{{< text bash >}}
+$ kubectl get istiorevisiontag
+{{< /text >}}
+
+{{< text plain >}}
+NAME      STATUS    IN USE   REVISION          AGE
+default   Healthy   True     default-v1-24-2   6m24s
+{{< /text >}}
+
+- Deploy the sample application
+
+{{< text bash >}}
+$ kubectl apply -f {{< github_file >}}/samples/sleep/sleep.yaml -n sample
+{{< /text >}}
+
+- Confirm istio proxy version of the sample app match the control plane version
+
+{{< text bash >}}
+$ istioctl proxy-status
+{{< /text >}}
+
+{{< text plain >}}
+NAME                              CLUSTER        CDS              LDS              EDS              RDS              ECDS        ISTIOD                                    VERSION
+sleep-5fcd8fd6c8-q4c9x.sample     Kubernetes     SYNCED (78s)     SYNCED (78s)     SYNCED (78s)     SYNCED (78s)     IGNORED     istiod-default-v1-24-2-bd8458c4-jl8zm     1.24.2
+{{< /text >}}
+
+#### Upgrade the Istio control plane to version 1.24.3
+
+- Update the `Istio` resource with the new version
+
+{{< text bash >}}
+$ kubectl patch istio default -n istio-system --type='merge' -p '{"spec":{"version":"v1.24.3"}}'
+{{< /text >}}
+
+- Check the `Istio` resource, you will see that there are two revisions and there are two `Istio` ready
+
+{{< text bash >}}
+$ kubectl get istio
+{{< /text >}}
+
+{{< text plain >}}
+NAME      REVISIONS   READY   IN USE   ACTIVE REVISION   STATUS    VERSION   AGE
+default   2           2       2        default-v1-24-3   Healthy   v1.24.3   10m
+{{< /text >}}
+
+- The `IstioRevisiontag` now references the new revision
+
+{{< text bash >}}
+$ kubectl get istiorevisiontag
+{{< /text >}}
+
+{{< text plain >}}
+NAME      STATUS    IN USE   REVISION          AGE
+default   Healthy   True     default-v1-24-3   11m
+{{< /text >}}
+
+- There are two `IstioRevisions`, one for each istio version
+
+{{< text bash >}}
+$ kubectl get istiorevision
+{{< /text >}}
+
+{{< text plain >}}
+NAME              TYPE   READY   STATUS    IN USE   VERSION   AGE
+default-v1-24-2          True    Healthy   True     v1.24.2   11m
+default-v1-24-3          True    Healthy   True     v1.24.3   92s
+{{< /text >}}
+
+The Sail Operator automatically detects whether a given Istio control plane is being used and writes this information in the "In Use" status condition that you see above. Right now, all `IstioRevisions` and our `IstioRevisionTag` are considered "In Use":
+* The old revision `default-v1-24-2` is considered in use because it is referenced by the sample application’s sidecar
```

Comment:
```suggestion
* The old revision `default-v1-24-2` is considered in use because it is referenced by the sample application’s sidecar.
```

Consistency.